Clean up Dark Mode borders around tables in Key Bindings & Advanced tabs

svobs opened the pull request:
I came up with this while working on #4107, but that's already too big, and wanted to get comments on this, so I pulled this out and put it into its own issue.

- [x] I have read [CONTRIBUTING.md](https://github.com/iina/iina/blob/develop/CONTRIBUTING.md)
- [ ] This implements/fixes issue #.

---

**Description:**

_[Updated from the original post: now much simpler - only needs border changes to 3 `NSBoxes` - but the changes here look a lot bigger because they are using the code from #4107 as a base...]_

I use Dark Mode for the most part, and it bothers me that the box colors don't match for the 3 tables in `Prefs > Key Bindings` and `Prefs > Advanced`. They seem ok in Light Mode. But in Dark Mode the boxes on the bottom (with the +/- buttons) are a dark gray, while the table above uses a light gray border, and although they aren't misaligned, the color difference makes them look like they are.

![Screenshot Old Light](https://user-images.githubusercontent.com/2213815/207597181-5c06569d-017c-47af-a0f2-ce53fbae7339.png)
![Screenshot Old Dark](https://user-images.githubusercontent.com/2213815/207597198-a6ae04be-e0d1-4631-a3ac-0455a7f3adad.png)

I first tried changing the borders of the scroll view to a darker color, so it matched the boxes below, but that looked a bit off. It looks like in most other parts of MacOS, the trend is to use light borders. XCode is a notable exception in that it seems to favor dark borders, but personally I don't find XCode very attractive or modern looking. Open to discussion though.

So I tried to find colors in the existing palette which worked for both light and dark, and `Separator Color` was the winner - in fact it looks like that is what modern `NSScrollView`s are using. So this PR just changes the border color for the box views below them to match.
![Screenshot Separator Color](https://user-images.githubusercontent.com/2213815/207598344-de42f078-00a5-4bb6-b138-9bd5dd5a88b0.png)

Results:
![Screenshot New Light](https://user-images.githubusercontent.com/2213815/207599406-26ff7d26-edc2-466d-8539-b17160750931.png)
![Screenshot New Dark](https://user-images.githubusercontent.com/2213815/207599414-24eb5b61-b484-4f09-8877-2e9f5c1d46b2.png)

Wanted to highlight this and see if anyone thinks this is a controversial change.

svobs commented:
Eh. Revisited this and decided to come to terms with deleting some endearing but unnecessary code. Revised the description appropriately and force-pushed a simpler commit.

svobs commented:
Resubmitted - previous push had some lingering XIB junk which was generating warnings

uiryuu commented:
Sorry for ignoring this for so long. I think the changes are good based on the image you posted. Could you redo this PR on the latest develop branch?

svobs committed:
Minor layout fixes to Key Bindings & Advanced boxes.

Changes:
• Change border color of NSBoxes below Key Bindings, Configuration, &
  MPV Options tables from "Grid" to "Separator Color", so that their
  borders are consistent with the tables above even in Dark Mode
• Adjust vertical offset of "Display raw values" checkbox to center it
  & add constraint to keep it aligned with "Duplicate…" button
• Remove focus ring from MPV Options table, so it matches other tables
  (in the future this table should be migrated from cell-based to
  view-based to best ensure consistency)

svobs commented:
Redid the PR. Made a couple other minor fixes which popped out at me. See the commit msg for details.

uiryuu commented:
Thanks!

low-batt commented:
This was included in IINA 1.4.0.
